src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerPushImage.groovy

```diff
@@ -47,49 +41,45 @@ class DockerPushImage extends AbstractDockerRemoteApiTask implements RegistryCre
 
     @Override
     void runRemoteCommand() {
-        PushImageCmd pushImageCmd = dockerClient.pushImageCmd(imageName.get())
+        tags.get().each { currentTag ->
+            logger.quiet "Pushing image with name '${currentTag}'."
```

Comment:
Maybe we should change _name_ to _tag_.

Reply:
Good idea. Will change that. Technically, `tag` is the wrong term here as well as we are talking about `<repository>/<image-name>:<tag>`. I don't think there's really a blanket term in Docker that would cover all of this.

Reply:
I would argue just to say `Pulling image ${currentImage}` (goes inline with comments I made above).

Reply:
done